css/style.css

```diff
@@ -9,400 +14,616 @@ body {
-        left: 60%;
-        height: 60%;
-    }
+.bag__title-bold {
+  position: absolute;
+  width: 400px;
+  height: 41px;
+  left: 55px;
+  top: 40px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
+  color: #000000;
+ }
+
+.bag__color {
+  position: absolute;
+  top: 375px;
+  left: 55px;
 }
 
-@media (max-width: 1270px) { 
-    .new-сollection__main-card {
-        width: 38%;
-        left: 60%;
-        margin-top: 5%;
-    }
+.color-wight,
+.color-blue,
+.color-green {
+  width: 30.18px;
+  height: 30px;
 }
-    
-@media (max-width: 1270px) { 
-    .girl {
-        width: 50%;
-        margin-left: 5%;
-    }
+
+.color-pink {
+  width: 38px;
+  height: 38px;
 }
 
-@media (max-width: 1100px) { 
-    .new-сollection__main-button {
-        width: 35%;
-    }
+.color-wight {
+  background:#DED;
 }
 
-@media (max-width: 1100px) { 
-    .new-сollection__main-link {
-        width: 50%;
-        margin-left: 3%;
-    }
+span {
+  position: absolute;
+  width: 125px;
+  height: 57px;
+  left: 288px;
+  top: 365px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 56px;
+  color: #000;
 }
 
-@media (max-width: 1100px) { 
-     .new-сollection-arrow {
-        width: 10%;
-    } 
-     }
+.button-minus,
+.button-plus {
+  border: none;
+}
 
+.button-number {
+  position: absolute;
+  width: 100px;
+  height: 44px;
+  left: 55px;
+  top: 468px;
+  border: 1.5px solid #000;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
 
-@media (min-width: 800px) { 
-    .logo-mobil,
-    .galka,
-    .new-collection__background_girl {
-        display: none;
-        } 
-    } 
+.bag__button {
+  position: absolute;
+  width: 253px;
+  height: 44px;
+  border: 1.5px solid #000;
+  box-sizing: border-box;
+  left: 170px;
+  top: 468px;
+  background: #FFF;
+}
 
-@media (max-width: 799px) { 
-    .logo,
-    .lang,
-    .check,
-    .new-сollection__social,
-    .text,
-    .new-сollection__main-card,
-    .new-collection__background {
-         display: none;
-        } 
-    } 
+.bag__link {
+  position: absolute;
+  width: 299px;
+  height: 58px;
+  margin-left: 884px;
+  top: 1663px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
+  color: #000;
+}
 
-@media (max-width: 800px) { 
-    .logo-mobil {
-          position: absolute;
-          width: 98px;
-          height: 37px;
-          padding: 5% 0 0 7%; 
-        }
-    }  
+.bag__arrow {
+  position: absolute;
+  width: 241px;
+  left: 1407px;
+  top: 1665px;
+}
 
-@media (max-width: 799px) { 
-    .girl {
-        background: none
-     }
-    }
+/* Медиа запросы */
 
-@media (max-width: 799px) { 
-    .new-сollection {
-        background: url(../img/girl2.jpg) no-repeat;
-        background-size: cover;
-        background-position: 0 92px; 
-        width: 80%;
-        margin-left: 10%;
-        position: absolute;
-        
+@media (max-width: 3000px) { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)